### PR TITLE
readme: add comparision table

### PR DESCRIPTION
**Pull request by JanTvrdik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,27 @@
...
 - 💡 **Easy Integration:** Simple to integrate with your existing Doctrine setup.
 
+
+## Comparison
+
+|                                                                        | Unoptimized | Manual Preload           | Fetch Join             | setFetchMode | EntityPreloader                |
+|------------------------------------------------------------------------|-------------|--------------------------|------------------------|--------------|--------------------------------|
+| [OneToMany](tests/EntityPreloadBlogOneHasManyTest.php)                 | 1 + n       | impossible in Doctrine 3 | 1, but duplicates rows | 1 + 1        | 1 + 1                          |
+| [OneToManyDeep](tests/EntityPreloadBlogOneHasManyDeepTest.php)         | 1 + n + n²  | impossible in Doctrine 3 | 1, but duplicates rows | 1 + 1 + n²   | 1 + 1 + 1                      |
+| [OneToManyAbstract](tests/EntityPreloadBlogOneHasManyAbstractTest.php) | 1 + n + n²  | impossible in Doctrine 3 | 1, but duplicates rows | 1 + 1 + n²   | 1 + 1 + 1, but duplicates rows |
```
Comment (janedbal) [inline README.md:16]: What about making all those `Ns` everywhere bold? Or somehow emphatize what is bad and what is good at first sight.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,27 @@
 - 🔄 **Flexible:** Supports all associations: `#[OneToOne]`, `#[OneToMany]`, `#[ManyToOne]`, and `#[ManyToMany]`.
 - 💡 **Easy Integration:** Simple to integrate with your existing Doctrine setup.
 
+
+## Comparison
+
+|                                                                        | Unoptimized | Manual Preload           | Fetch Join             | setFetchMode | EntityPreloader                |
```
Comment (janedbal) [inline README.md:12]: Unoptimized -> Default

Comment (janedbal) [reply README.md:12]: What about linking specific slides for each header title? Might be hard to uderstand each approach without seeing it.

File: README.md (hunk 1 above)
Comment (JanTvrdik) [reply README.md]: I adding emoji. Does that help?

Comment (janedbal) [reply README.md]: Yes!

**Review by janedbal — APPROVED** [review]

**Pull request merged**